test/parallel/test-runner-run.mjs

```diff
@@ -68,10 +69,9 @@ describe('require(\'node:test\').run', { concurrency: true }, () => {
     const result = await run({
       files: [join(testFixtures, 'default-behavior/test/random.cjs')]
     }).compose(dot).toArray();
-    assert.deepStrictEqual(result, [
-      '.',
-      '\n',
-    ]);
+
```

Comment:
```suggestion
assert.strictEqual(result.length, 2);
```

I'm not sure if this is even needed, would there be a case where the length isn't two?

Reply:
@RedYetiDev I really don't think so... it was already there, I can remove it

Reply:
@RedYetiDev sure? I can add it, I don't think it is super valuable tho

Reply:
Whatever you think is best

Reply:
@RedYetiDev well, updated 😄